Lib/test/test_faulthandler.py

```diff
@@ -140,7 +136,7 @@ def test_read_null(self):
                 3,
                 'access violation')
 
-    @requires_raise
+    @unittest.skipIf(is_android, 'raising SIGSEGV on Android is unreliable')
```

Comment:
Please write a function like the current requires_raise() for Android, since you copied/pasted this line many times. Something like:

```
def skip_on_android():
    return unittest.skipIf(is_android, 'raising SIGSEGV on Android is unreliable')
```

Maybe "skip_segfault_on_android"?

Add a comment on this function mentionning "bpo-32138" and explain that raise(SIGSEGV) doesn't crash on Android (but does nothing if I understood correctly).

Then just use @skip_on_android.